uk.co.compucorp.civicrm.tasksassignments/js/src/crm-tasks-workflows/controllers/casetype.extended.controller.js

```diff
@@ -53,6 +54,48 @@
       $scope.caseType.definition.activitySets[0].activityTypes = [];
     }
 
+    /**
+     * Returns the default relationship type options. If the relationship is
+     * bidirectional (Ex: Spouse of) it adds a single option otherwise it adds
+     * two options representing the relationship type directions
+     * (Ex: Employee of, Employer is)
+     *
+     * @return {Array}
+     */
+    function getDefaultRelationshipTypeOptions () {
+      var activeRelationshipTypes = getActiveRelationshipTypes();
+      var defaultRelationshipTypeOptions = [];
+
+      activeRelationshipTypes.forEach(function (relationshipType) {
+        var isBidirectionalRelationship = relationshipType.label_a_b === relationshipType.label_b_a;
```

Comment:
Are **labels** safe to compare? Just a question, not a CTA.

Reply:
This is the exact same question I made to core. I personally believe it's not safe since they can be changed by the user, but that's what they recommended and what they are using internally. 

Please remember that I'm trying to override a function that actually exists in core, so I'm trying to follow it as closely as possible.

This is my original question and their answer follows soon after:
https://github.com/civicrm/civicrm-core/pull/11998#issuecomment-390243451

At that time, I also double checked and asked through MatterMost.

Reply:
Please check if labels can be duped. If not, I would say it is **acceptable** to compare them then.